NewForce-Cohort-5/tabloidfullstack-actual-spring
# Jt commentlist

# Description

A user should be able to view comments that are related to a post. Upon viewing a post's details a user can select view comments to be directed to the list of comments for the post with the title of the post above. This list should include the users display name, subject, content, and short date string. For now you will need to change the URL to access the location of the list.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. After doing a `git add` `git commit` on your current branch checkout to the main branch and fetch all of the branches.
2. Checkout to `jt-commentList` run your server, and log into the app using Foo@bar.com
3. once you are logged in add /post/comments/1 to your browser URL and you should be redirected to the post with an ID of 1
4. Please test a few more times using different post IDs and verify the property related information



# Checklist:
